crates/core/src/piop/tests.rs

```diff
@@ -152,15 +152,11 @@ fn commit_prove_verify<FDomain, FEncode, F, P, MTScheme>(
 	let mut proof = ProverTranscript::<HasherChallenger<Groestl256>>::new();
 	proof.message().write(&commitment);
 
-	let hal = CpuLayer::<F>::default();
-	let mut host_mem = vec![F::ZERO; 1 << 14];
-	let mut dev_mem = vec![F::ZERO; 1 << 22];
+	let mut compute_holder = CpuLayerHolder::<F>::new(1 << 14, 1 << 22);
```

Comment:
I was expecting that these test will also be generic in compute layer, so that we can also test them from ic1. What is the plan around that?